pkg/users/account-create-dialog.js

```diff
@@ -66,10 +67,20 @@ function AccountCreateBody({ state, errors, change }) {
                                 idPrefix="accounts-create-password"
                                 change={change} />
 
-            <FormGroup label={_("Access")} fieldId="accounts-create-locked" isInline>
+            <FormGroup label={_("Options")} fieldId="accounts-create-locked" hasNoPaddingTop>
                 <Checkbox id="accounts-create-locked"
-                          label={_("Lock account")}
-                          isChecked={locked} onChange={checked => change("locked", checked)} />
+                          isChecked={locked} onChange={checked => change("locked", checked)}
+                          label={
+                              <>
+                                  {_("Disallow password usage")}
+                                  <Popover bodyContent={_("Other authentication methods are still available even when password authentication is disallowed")}>
```

Comment:
```suggestion
                                  <Popover bodyContent={_("Other authentication methods are still available even when password authentication is disallowed.")}>
```

This popover has a sentence. It needs a period.